contracts/acl/ACL.sol

```diff
@@ -336,15 +395,17 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
         Param memory param = permissionParams[_paramsHash][_paramId];
 
         if (param.id == LOGIC_OP_PARAM_ID) {
-            return _evalLogic(param, _paramsHash, _who, _where, _what, _how);
+            return (Op(param.op) == Op.IF_ELSE)
```

Comment:
I don’t fully get the rationale behind this change either.

Reply:
Many of these changes were due to avoid "stack too deep" issues